docs/source/userguide/function_list.rst

```diff
@@ -1,13 +0,0 @@
-.. dropdown:: Distribution Statement
-
- | # # # This source code is protected under the license referenced at
- | # # # https://github.com/NRLMMD-GEOIPS.
-
-********************
-GeoIPS Functionality
-********************
-
-The GeoIPS Command Line Interface is currently under development -
```

Comment:
Is this on the radar to be automated in the documentation with the CLI?

Reply:
? Do you mean removing blank files?